src/init/global.init.s

```diff
@@ -4,20 +4,34 @@
 global_init:
 	ld hl,global_.sheets
 	ld b,n_sheets
-@sheets_loop:
-;	for now they are all rocks
+
+; ////// ROCK \\\\\\
 	ld a,%10010000	; size = 0, block, hurt by bombs.
 	ldi (hl),a
 	ld a,0	; dmg = 0
 	ldi (hl),a
 	ld a,0
 	ldi (hl),a
 	ldi (hl),a ; no funtion pointer or whatever
+	ldi (hl),a ; speed = 0
+	ld a,0
+	ldi (hl),a ; 0 HP
+; \\\\\\ ROCK //////
+
+; ////// VOID \\\\\\
+	ld a,%00010000	; size = 0, block, hurt by bombs.
+	ldi (hl),a
+	ld a,0	; dmg = 0
+	ldi (hl),a
+	ld a,0

```

Comment:
same

Reply:
same, it just happens to be 0, but we may want to change it.